Source/SwiftLintFramework/Models/SwiftVersion.swift

```diff
@@ -23,4 +23,11 @@ enum SwiftVersion {
 
         fatalError("Unexpected Swift version")
     }()
+
+    var nameKey: String {
```

Comment:
This is now used in just one place, keep it there? This shouldn't be a widely used thing.